uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/components/my-leave-report.js

```diff
@@ -155,6 +156,10 @@ define([
       $rootScope.$on('LeaveRequest::new', function () {
         vm.refresh();
       });
+
+      $rootScope.$on('LeaveRequest::edit', function () {
```

Comment:
Can you group all the `$rootScope.$on` calls in a separate function? Would make the code better organized